### PR TITLE
Aida 1287

### PR DESCRIPTION
**Pull request by sharknc**
- Reviewed AWS Batch documentation 
- Added missing configurations to the batch initializer README.md
- Added missing documentation on creating custom AMI's to deploy Batch Single docker containers on within AWS batch

### CODE REVIEW

File: docker/batch-single/README.md [hunk 1]
```diff
@@ -79,7 +79,10 @@ $ chmod +x run.sh
 $ ./run.sh
 ```
 
+### Compute Environment AMI
```
Comment (nextcen-dgemoets) [inline docker/batch-single/README.md]: Do `WORKER_INIT_TIMEOUT`, `AWS_SNS_TOPIC_ARN`, and `DEBUG_TIMEOUT` need to be added to the variables section above?  They are in `run.sh.example`.  If not, let me know and I'll approve the PR.

Comment (sharknc) [reply docker/batch-single/README.md]: Good catch! I was actually missing a few others as well. I have updated the README.md to include all possible configurations.

File: docker/batch-init/README.md [hunk 2]
```diff
@@ -99,7 +102,10 @@ To run the Batch Scheduler Docker container, copy the `run-scheduler.sh.example`
 | `BATCH_NUM_NODES`        	 | The number of nodes to deploy the AWS Batch job on. *This must match the number of nodes entered when deploying the AWS Batch infrastructure via CloudFormation |  
 | `BATCH_JOB_DEFINITION`       | The AWS Batch job definition to use when executing the validation job |  
 | `BATCH_JOB_QUEUE`     		 | The AWS Batch job queue to use when executing the validation job |  
-| `AWS_SNS_TOPIC_ARN`     	 | The AWS SNS topic to push notifications to during the AWS Batch validation job |  
+| `AWS_SNS_TOPIC_ARN`     	 | The AWS SNS topic to push notifications to during the AWS Batch validation job |
+| `AWS_DEFAULT_REGION`       | The default AWS region to use during the AWS Batch validation job |
+| `NIST_VALIDATION_FLAG`     | The validation flags to pass to the validator, e.g., `--ldc --nist -o --disk` |
+| `JAVA_OPTS`                | The Java options to pass to the AWS Batch validation job, e.g, `-Xmx10G'` |
```
Comment (nextcen-dgemoets) [inline docker/batch-init/README.md]: Another minor thing-- could you fix the `the the` in the `Setting up Batch Scheduler Cron Job` section?

File: docker/batch-single/README.md [hunk 3]
```diff
@@ -65,12 +65,18 @@ To run the Batch Single Docker container, copy the `run.sh.example` script to a
 | `S3_SUBMISSION_PREFIX`     	 | The prefix (directory) where the extracted submission ttl files are located | 
 | `S3_SUBMISSION_VALIDATION_DESCR` | The description of the validation type that will be performed on the submission |       
 | `AWS_BATCH_JOB_ID`       | The job ID assigned for this validation from AWS Batch | 
-| `AWS_BATCH_JOB_NODE_INDEX`       | The node ID assigned to current node from AWS Batch |    
-| `MAIN_SLEEP_INTERVAL`        	 | Interval for the main to monitor the SQS queue for depletion |  
+| `AWS_BATCH_JOB_NODE_INDEX`       | The node ID assigned to current node from AWS Batch |
```
Comment (nextcen-dgemoets) [inline docker/batch-single/README.md]: Another minor thing-- could you fix the `The The` and the `Validatior` typos in the `Batch Single` section above?

Comment (sharknc) [reply docker/batch-single/README.md]: Updated.

**Pull request merged**